File: 58_length_of_last_word.py
class Solution:
    def lengthOfLastWord(self, s: str) -> int:
        memory = ""
        for n in range(len(s)):
            if s[n] != " ":
                if s[n - 1] == " ":
                    memory = s[n]
                else:
                    memory += s[n]
        return len(memory)

File: 58_length_of_last_word.cpp
class Solution {
public:
    int lengthOfLastWord(string s) {
        std::string memory = "";
        for (int n = 0; n < s.size(); n++) {
            if (s[n] != ' ') {
                if (n != 0 && s[n - 1] == ' ') {
                    memory = s[n];
                } else {
                    memory += s[n];
                }
            }
        }
        return memory.size();
    }
};